localstack/aws/connect.py

```diff
@@ -289,11 +290,11 @@ def _get_client(
         service_name: str,
         region_name: str,
         use_ssl: bool,
-        verify: bool,
-        endpoint_url: str,
-        aws_access_key_id: str,
-        aws_secret_access_key: str,
-        aws_session_token: str,
+        verify: Optional[bool],
+        endpoint_url: Optional[str],
+        aws_access_key_id: Optional[str],
+        aws_secret_access_key: Optional[str],
+        aws_session_token: Optional[str],
```

Comment:
I think it's a good idea to keep these args explicit as they form part of the cache key

Reply:
You'll still need to provide them in this case since they don't have default values. In some cases (e.g. against AWS) we need to set them to None for boto to set them automatically. The None value in this case will just be part of the cache key